diff --git a/node_modules/@nuxt/content/dist/runtime/server/search.js b/node_modules/@nuxt/content/dist/runtime/server/search.js
index 46e1c32..74f2bac 100644
--- a/node_modules/@nuxt/content/dist/runtime/server/search.js
+++ b/node_modules/@nuxt/content/dist/runtime/server/search.js
@@ -14,7 +14,7 @@ export function splitPageIntoSections(page, { ignoredTags }) {
     id: path,
     title: page.title || "",
     titles: [],
-    content: page.description.trim() || "",
+    content: (page.description|| "").trim(),
     level: 1
   }];
   if (!page?.body?.children) {
